Content.Shared/Weapons/Reflect/ReflectComponent.cs

```diff
@@ -16,6 +17,25 @@ public sealed partial class ReflectComponent : Component
     [ViewVariables(VVAccess.ReadWrite), DataField]
     public ReflectType Reflects = ReflectType.Energy | ReflectType.NonEnergy;
 
+    /// <summary>
+    /// Select in which inventory slots it will reflect.
+    /// By default, it will reflect in any inventory position, except pockets.
+    /// </summary>
+    [ViewVariables(VVAccess.ReadWrite), DataField]
```

Comment:
```suggestion
    [DataField]
```
Datafield already includes VV readwrite nowadays.
Same below.

Reply:
done